cmake/SSGCommon.cmake

```diff
@@ -167,7 +168,8 @@ macro(ssg_build_oval_unlinked PRODUCT)
         add_custom_command(
             OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/oval-unlinked.xml
             # TODO: config
-            COMMAND RUNTIME_OVAL_VERSION=5.11 ${SSG_SHARED_UTILS}/combine-ovals.py ${CMAKE_SOURCE_DIR}/config ${PRODUCT} ${OVAL_DEPS_DIR} ${OVAL_511_DEPS_DIR} ${SHARED_OVAL_DEPS_DIR} ${SHARED_OVAL_511_DEPS_DIR} > ${CMAKE_CURRENT_BINARY_DIR}/oval-unlinked.xml
+            COMMAND SHARED=${SSG_SHARED} ${SSG_SHARED_UTILS}/generate-from-templates.py --oval_version ${OSCAP_OVAL_VERSION} --input ${CMAKE_CURRENT_SOURCE_DIR}/templates --output ${CMAKE_CURRENT_BINARY_DIR}/build --language oval build
```

Comment:
I'd build remediations into CURRENT_BINARY_DIR/remediations/$language and OVAL into CURRENT_BINARY_DIR/oval

CURRENT_BINARY_DIR/build looks like a typo. I would end up as `git_root/build/RHEL/7/build`